packages/components/src/products.tsx

```diff
@@ -238,53 +253,49 @@ export const SIX_HIGHLIGHTED_PRODUCTS = [
 ];
 
 /** List of products displayed in hamburger menu. */
-export const PRODUCTS_MENU_LIST: Record<string, { type: 'separator'; title: ReactNode }> =
-  Object.fromEntries(
-    (
-      [
-        'The GraphQL Stack',
-        ...FOUR_MAIN_PRODUCTS,
-        'Libraries',
-        ...SIX_HIGHLIGHTED_PRODUCTS,
-      ] as const
-    ).map((item, i) => {
-      if (typeof item === 'string') {
-        return [
-          i,
-          {
-            type: 'separator',
-            title: (
-              <>
-                {/* This is a one-off class. The margins and paddings of the parent list item are were large. */}
-                {/* eslint-disable-next-line tailwindcss/no-custom-classname */}
-                <style className="hive-label-separator">
-                  {'li:has(>.hive-label-separator) { margin: 0.75rem 0 0.25rem 0; padding: 0 }'}
-                </style>
-                <span className="ml-2 font-medium text-gray-500 dark:text-neutral-400">{item}</span>
-              </>
-            ),
-          },
-        ];
-      }
+export const PRODUCTS_MENU_LIST = Object.fromEntries(
+  (
+    ['The GraphQL Stack', ...FOUR_MAIN_PRODUCTS, 'Libraries', ...SIX_HIGHLIGHTED_PRODUCTS] as const
+  ).map((item, i) => {
+    if (typeof item === 'string') {
       return [
         i,
         {
-          type: 'page',
-          href: item.href,
+          // BUG: Nextra 4 doesn't support separators inside menu items.
+          href: '#',
```

Comment:
not a bug, but separators were never supported in Nextra 2/3 in `type: menu`, but custom themes can support it, can you add support of it? https://github.com/shuding/nextra/blob/755ecd09d2a26470573363c8141589dfae5db2a8/packages/nextra/src/server/schemas.ts#L148